docs/customize/authentication.md

```diff
@@ -1007,7 +1007,7 @@ Another possible scenario, not yet supported, could be to grant or restrict acce
 such as records or files.
 
 The support of groups is a feature  introduced in the release
-[v9.0](https://inveniordm.docs.cern.ch/releases/versions/version-v9.0.0/).
+[v9.0](https://inveniordm.docs.cern.ch/releases/v9/version-v9.0.0/).
```

Comment:
Thanks for that! Let's use this link instead: 

```suggestion
[v9.0](../releases/v9/version-v9.0.0.md).
```

When links are defined this way, if they don't resolve, a warning will be emitted by the documentation building step which will fail the doc tests. This way, when that content (or the referring page) changes location, we will be warned/prevented from going live with it. (i.e., this link won't slip past us again!)